static/scss/partials/header.scss

```diff
@@ -349,7 +372,7 @@ header {
     }
 }
 
-@media screen and (max-width: 550px) {
+@media screen and (max-width: 690px) {
```

Comment:
Had to increase the media query by a `10px` magic number to prevent line breaks in the buttons.

Reply:
This works but I wanted to call out the fact that l10n will mean line breaks will totally happen (and it's okay!)